app/code/Magento/CatalogImportExport/Model/Export/Product.php

```diff
@@ -691,7 +691,7 @@ protected function updateDataWithCategoryColumns(&$dataRow, &$rowCategories, $pr
     /**
      * {@inheritdoc}
      */
-    public function _getHeaderColumns()
+    public function getHeaderColumns()
```

Comment:
Changing the name of the public method is backward incompatible, please, follow [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) for more information.

Reply:
I did the check in the source and based on the comment that in the future release _getHeaderColumns will me used, it didnt occur to me that it might be widely used throughout the code base already.